onwuvic/MyDiary
# #158910063 Reminder settings interface

#### What does this PR do?
Reminder settings interface. A user can set if they want a reminder or not.

#### Description of Task to be completed?
As a User, I should be able to check a button under settings, so that I can decide if I want a daily reminder or not

#### How should this be manually tested?
Given a user
When User navigates to reminder settings
Then User should see a check a button.

#### What are the relevant pivotal tracker stories?
#158910063
